its/projects/TwoDrivesThreeProjects/TwoDrivesThreeProjects.sln

```diff
@@ -0,0 +1,37 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.2.32526.322
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DefaultDriveFirstProject", "DefaultDrive\DefaultDriveFirstProject.csproj", "{5B64B1C0-C68F-4585-B02A-8789CD7C31CE}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DriveY", "Y:\DriveY.csproj", "{1B0529BA-29F3-49F1-9977-99050FAC907A}"
```

Comment:
Maybe add a small `readme.md` next to this SLN explaining the purpose of `setup.bat` and `cleanup.bat`. Because opening the SLN doesn't find the `DriveY` - that is correct behavior that we understand now, but others might not understand what the hell is happening

Reply:
yes, this is why I wanted to have these scripts together with the solution. I'll add a readme